apps/web/src/app/(private)/_data-access/index.ts

```diff
@@ -173,6 +173,15 @@ export const getEvaluationByUuidCached = cache(async (uuid: string) => {
   return evaluation
 })
 
+export const getEvaluationByIdCached = cache(async (id: number) => {
+  const { workspace } = await getCurrentUser()
+  const evaluationScope = new EvaluationsRepository(workspace.id)
+  const result = await evaluationScope.findById(id)
```

Comment:
all repos implement `find` method which is ==  to this `findById` so we can probably remove this one